x-pack/plugin/security/qa/security-basic/src/test/java/org/elasticsearch/xpack/security/CatIndicesWithSecurityIT.java

```diff
@@ -0,0 +1,126 @@
+/*
+ *
+ *  * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ *  * or more contributor license agreements. Licensed under the Elastic License;
+ *  * you may not use this file except in compliance with the Elastic License.
+ *
+ */
+
+package org.elasticsearch.xpack.security;
```

Comment:
I'm not sure if this is the right place to put this file, but it should be easy enough to move.

Reply:
Hmm. I think it might be better placed under `x-pack/plugin/src/test` since it doesn't relate specifically to security with a basic license.